Handler/Events.hs

```diff
@@ -58,14 +58,26 @@ addListMetaData urlRender totalCount keyValues =
             ]
 
 
+textToOrder :: Text -> SelectOpt Event
+textToOrder text =
+    case textWithNoPrefix of
+        "id"    -> direction text $ EventId
+        "title" -> direction text $ EventTitle
+        "user"  -> direction text $ EventUserId
+        _       -> error "Wrong order"
+
+    where textWithNoPrefix = if T.isPrefixOf "-" text
+                then T.tail text
+                else text
+          direction t = if T.isPrefixOf "-" t
```

Comment:
we could, but then we need to cast from Text to String. And anyway `T.isPrefixOf "-"` is short enough :)